web/locales/fi.json

```diff
@@ -0,0 +1,1786 @@
+        "mapArrows-answer": "Alueiden väliset nuolet osoittavat sähkön fyysisen virtauksen (tuonnit ja viennit) alueiden välillä: mitä nopeammin ne vilkkuvat, sitä nopeampi virtaus on. Sähkövirta on tärkeä, koska kun tuot osaa sähköstä naapurialueilta, tuot myös sen tuotantoon liittyviä hiilidioksidipäästöjä.",
+        "mapSolarWindButtons-question": "Mitä kartalla olevat aurinko- ja tuulipainikkeet tekevät?",
+        "mapSolarWindButtons-answer": "Nämä painikkeet näyttävät reaaliaikaisen tuulen nopeuden ja auringonpaisteen voimakkuuden, mikä osoittaa mahdollisuuden siirtyä aurinko- ja tuulipohjaisiin energiavaihtoehtoihin eri alueilla. Huomaa, että tämä on vain pintakäsittely, koska tuuliturbiinien ja aurinkokennojen asentamisen todellinen potentiaali riippuu myös monista muista tekijöistä.",
+        "mapNuclearColors-question": "Miksi ydinkäyttöiset maat näkyvät kartalla vihreinä?",
+        "mapNuclearColors-answer": "Ydinenergian tuotannossa on erittäin alhainen hiili-intensiteetti (<a href=\"https://en.wikipedia.org/wiki/Life-cycle_greenhouse-gas_emissions_of_energy_sources#2014_IPCC.2C_Global_warming_potential_of_selected_electricity_sources\" target=\"_blank\" >lähde</a>), ja ydinvoimaloiden maat esitetään siksi kartalla vihreinä. Tämä ei tarkoita, että ydinenergiaan ei liity muita ympäristöhaasteita (kuten myös muissa energiantuotantomuodoissa), mutta nämä haasteet eivät liity kasvihuonekaasujen päästöihin, joten ne eivät kuulu tämän kartan laajuteen.",
+        "mapColorBlind-question": "Olen värisokea. Kuinka voin lukea karttaasi?",
+        "mapColorBlind-answer": "Voit muuttaa kartassa näkyviä värejä ottamalla käyttöön ”värisokamoodi” -asetuksen. Se sijaitsee työpöydän vasemman paneelin alaosassa ja mobiilisovelluksen tietovälilehdessä."
+    },
+    "mapAreas": {
+        "groupName": "Kartan alueet",
+        "noData-question": "Miksi alueeni tietoja ei ole saatavilla?",
+        "noData-answer": " Joko siksi, että alueen tietolähteet eivät ole väliaikaisesti käytettävissä, tai koska meillä ei ole vielä mitään tietolähteitä alueelle. Voit <a href=\"#contribute\" class=\"entry-link\">auttaa meitä lisäämällä tietolähteitä</a>.",
+        "whySmallAreas-question": "Miksi jaat maailman pienemmille alueille, etkä näytä vain maiden keskiarvoja?",
+        "whySmallAreas-answer": "Alueet ovat pienimmät maantieteelliset alueet, joista meillä on tietoja. Antamalla tiedot yksityiskohtaisesti, sähkönkuluttajat eri alueilla voivat saada täsmällistä tietoa kulutetun sähkön alkuperästä ja siihen liittyvistä ilmastovaikutuksista.",
+        "divideExistingArea-question": "Voitko jakaa alueeni pienempiin osiin?",
+        "divideExistingArea-answer": "Toki, jos alueen jokaiselle osalle on olemassa erilliset tietolähteet. Voit <a href=\"#contribute\" class=\"entry-link\">auttaa meitä lisäämällä uusia tietolähteitä</a>",
+        "seeHistoricalData-question": "Voinko nähdä alueen historian, joka on kauempana kuin 24 tuntia?",
```

Comment:
I think `vanhempi` would be better than `kauempana`, kauempana is mostly used when talking about distance